machinist/mserver/flags.go

```diff
@@ -0,0 +1,23 @@
+package mserver
+
+import "github.com/enfabrica/enkit/machinist"
+
+type Modifier func(s *server) error
+
+func WithMachinistFlags(mods ...machinist.Modifier) Modifier {
+	return func(s *server) error {
+		for _, mod := range mods {
+			if err := mod(s); err != nil {
+				return err
+			}
+		}
+		return nil
+	}
+}
+
+func WithController(controller *Controller) Modifier {
```

Comment:
why does a `machinist server` require knowing a controller?

Reply:
It doesn't. It populates the server with an empty one at the start. It gives the option of passing one in however to introspect state e.g. for testing or boiling up scope.

Reply:
It doesn't. It populates the server with an empty one at the start. It gives the option of passing one in however to introspect state e.g. for testing or boiling up scope.